src/lib.rs

```diff
@@ -99,6 +103,42 @@
 //!
 //! For simple but complete examples, see the [examples] directory.
 //!
+//! ## Usage with [`actix_multipart`]
+//! <div class="warning">
+//!
+//! This requires the `actix-multipart` feature to be enabled
+//!
+//! </div>
+//!
+//! Usage with [`actix_multipart`] is almost the same as using it with
+//! actix's buiiltin [`Form`](actix_web::web::Form), with one exception:
+//! the `csrf_token` field must have type [`Text<CsrfToken>`](actix_multipart::form::text::Text)
+//! to interpret the CSRF token from the
+//! multipart form data. This works as follows:
+//!```
+//! # use actix_web::{HttpResponse, Responder, post};
+//! use actix_multipart::form::{text::Text, MultipartForm};
+//! use actix_csrf::extractor::{Csrf, CsrfGuarded, CsrfToken};
```

Comment:
Oh yeah I kinda forgot about that. I usually don't like doing that because it makes the docs hard to read inline, but you're right kn that it makes more sense to do so.